tox.ini

```diff
@@ -2,16 +2,19 @@
 skipsdist=True
 usedevelop=True
 
-envlist=dj{18}-{sqlite,postgres}
+envlist=dj{18,19,110,111}-{sqlite,postgres}
```

Comment:
I wasn't sure what to include here. Which versions should I keep in the tox.ini file?